src/components/file-input/UIFileInput.vue

```diff
@@ -42,15 +43,21 @@
                                transition-all rounded-l text-color disabled:cursor-not-allowed
                                bg-gray-200 dark:bg-gray-700 ring-gray-400 dark:ring-gray-500"
                         :style="{
-                            'filter': disabled ? 'invert(5%)': 'invert(0%)'
+                            'filter': disabled ? 'invert(5%)': 'invert(0%)',
+                            'border': light
+                             ? error || $slots.error ? '1px solid #B91C1CFF' : '1px solid #CFD8E3'
```

Comment:
if possible avoid hardcoded colours. Either use classes or if push comes to shove use the `var()` with the correct variable for your colour